configuration/structure.json

```
@@ -1,889 +1,1164 @@
+                            "running": {
+                                "friendlyName": "Running",
+                                "children": {
+                                    "runningWorkspace": { "friendlyName": "Running a Workspace", "children": {}, "content": "resources/Editor/Running/Running_Workspace" },
+                                    "runningProject": { "friendlyName": "Running a Project", "children": {}, "content": "resources/Editor/Running/Running_Project" }
+                                },
+                                "content": "landing_pages/extensionsEditorRunningLandingPage"
                             },
-                            "content": "extensions/WebGPU/webGPUInternals"
-                        }
-                    },
-                    "content": "extensions/WebGPU/webGPU"
-                },
-                "serverSide": {
-                    "friendlyName": "Running Babylon.js on a Server",
-                    "children": {},
-                    "content": "landing_pages/Support/nullEngine"
-                }
-            },
-            "content": "landing_pages/advanced_topics_landing_page"
-        },
-        "toolsAndResources": {
-            "friendlyName": "Tools and Resources",
-            "children": {
-                "tools": {
-                    "friendlyName": "Tools",
-                    "children": {
-                        "playground": {
-                            "friendlyName": "Playground",
-                            "children": {
-                                "externalPGAssets": { "friendlyName": "Using External Assets In The Playground", "children": {}, "content": "resources/Playground_aspects/External_PG_assets" },
-                                "pgTemplates": { "friendlyName": "Playground Templates", "children": {}, "content": "resources/Playground_aspects/playgroundTemplates" },
-                                "yourOwnSnippetServer": {"friendlyName": "Make your own Snippet Server", "children": {}, "content": "resources/Playground_aspects/yourOwnSnippetServer"}
+                            "assets": {
+                                "friendlyName": "Assets",
+                                "children": {
+                                    "addingMeshes": { "friendlyName": "Adding Meshes", "children": {}, "content": "resources/Editor/Assets/Adding_Meshes" },
+                                    "addingMaterials": { "friendlyName": "Adding Materials", "children": {}, "content": "resources/Editor/Assets/Adding_Materials" },
+                                    "addingTextures": { "friendlyName": "Adding Textures", "children": {}, "content": "resources/Editor/Assets/Adding_Textures" },
+                                    "addingSounds": { "friendlyName": "Adding Sounds", "children": {}, "content": "resources/Editor/Assets/Adding_Sounds" }

```

```suggestion
                                    "addingSounds": { "friendlyName": "Adding Sounds", "children": {}, "content": "resources/Editor/Assets/Adding_Textures" }
```
The file is missing, temporarily setting to a different file